lib/internal/modules/cjs/loader.js

```diff
@@ -1250,47 +1255,49 @@ let Script;
  * @param {object} codeCache The SEA code cache
  */
 function wrapSafe(filename, content, cjsModuleInstance, codeCache) {
+  const hostDefinedOptionId = Symbol(`cjs:${filename}`);
+  async function importModuleDynamically(specifier, _, importAssertions) {
+    const cascadedLoader = getCascadedLoader();
+    return cascadedLoader.import(specifier, normalizeReferrerURL(filename),
+                                 importAssertions);
+  }
   if (patched) {
-    const wrapper = Module.wrap(content);
-    if (Script === undefined) {
-      ({ Script } = require('vm'));
-    }
-    const script = new Script(wrapper, {
-      filename,
-      lineOffset: 0,
-      importModuleDynamically: async (specifier, _, importAssertions) => {
-        const cascadedLoader = getCascadedLoader();
-        return cascadedLoader.import(specifier, normalizeReferrerURL(filename),
-                                     importAssertions);
-      },
-    });
+    const wrapped = Module.wrap(content);
+    const script = makeContextifyScript(
+      wrapped,                 // code
```

Comment:
nit: what kind of code? numeric code? user code?

Reply:
It’s the source code of the module.

Reply:
Yes, source code, this has been part of the documentation for all the compilation APIs, so this PR is probably not the right place to change it..